src/pepr/uds-cluster-crds/templates/clusterconfig.uds.dev.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    meta.helm.sh/release-name: zarf-1480d00ca85972047319d79d2ea7253325be2d4c
+    meta.helm.sh/release-namespace: default
+  labels:
+    app.kubernetes.io/managed-by: Helm
```

Comment:
I'm not sure these should be here? Especially since we explicitly set them in the action, might be candidates to delete in the `yq` part of the task?